### PR TITLE
Enh snake

### PR DESCRIPTION
**Pull request by tacaswell**
attn @imrealkaren

### CODE REVIEW

File: bluesky/callbacks/core.py [hunk 1]
```diff
@@ -534,13 +534,17 @@ def start(self, doc):
         self.im = im
         self.ax.set_title('scan {uid} [{sid}]'.format(sid=doc['scan_id'],
                                                       uid=doc['uid'][:6]))
+        self.snaking = doc.get('snaking', (False, False))
```
Comment (danielballan) [inline bluesky/callbacks/core.py]: Should this be `doc.get('plan_args', {}).get('snaking', (False, False))`?

Comment (tacaswell) [inline bluesky/callbacks/core.py]: I think `snaking` goes in both places.

Comment (danielballan) [inline bluesky/callbacks/core.py]: I'm 60% sure you are wrong. Code link?

Comment (danielballan) [inline bluesky/callbacks/core.py]: ...Or, you know, a test :imp: ... but I would accept a link to the relevant line in plans.py where this happens.

Comment (tacaswell) [inline bluesky/callbacks/core.py]: https://github.com/NSLS-II/bluesky/blob/master/bluesky/plans.py#L782

Comment (tacaswell) [inline bluesky/callbacks/core.py]: Even better...with blame https://github.com/NSLS-II/bluesky/blame/master/bluesky/plans.py#L782 :smiling_imp:

Comment (danielballan) [inline bluesky/callbacks/core.py]: :100: 

When I write code that I can't even recall, I know I'm on my way to @tacaswell levels of glory. (Thanks for obliging me by digging this up.)

**Pull request merged**